components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -11,6 +11,8 @@
 namespace scream
 {
 
+ using namespace p3;
```

Comment:
This should still stay inside the class declaration. No need to inject all of p3's names in the scream namespace.

Reply:
Putting it inside the class declaration causes an error: "error: expected an identifier
    using namespace p3;"
To resolve this I removed the line "using namespace p3;" and just added the p3 namespace as a prefix to the needed lines:
line 24 of atmosphere_microphysics.hpp:
using P3F          = p3::Functions<Real, DefaultDevice>; 
and line 214 of atmosphere_microphysics.cpp:
p3::p3_init();